src/lib/test-handler.js

```diff
@@ -65,6 +68,22 @@ export default class TestHandler {
       });
   }
 
+  mergeLogs() {
+    let testResults = [];
+    let mergedFileName = path.join(this.options.logDir, 'merged', 'results.json');
+    let logFilePaths = fs.readdirSync(this.options.logDir);
+    logFilePaths.forEach((logFilePath) => {
+      if (_.endsWith(logFilePath, '.json')) {
+        testResults = _.concat(
+          testResults,
+          JSON.parse(fs.readFileSync(path.join(this.options.logDir, logFilePath), 'utf8'))
```

Comment:
Interesting, yeah sounds like requiring json is bad practice indeed. Thanks for the heads up